platform/iphone/in_app_store.mm

```diff
@@ -189,11 +192,9 @@ - (void)paymentQueue:(SKPaymentQueue *)queue updatedTransactions:(NSArray *)tran
 				int sdk_version = 6;
 
 				if ([[[UIDevice currentDevice] systemVersion] floatValue] >= 7.0) {
```

Comment:
Do we need ancient code for iOS 7?

Reply:
We don't need it, but it's still exists in `4.0`. And I planned to remove it later PRs to keep this PR as simple as possible. I'll remove it required. :)

Reply:
Is it possible to build it using iOS 7 target? If it's not (with is definitely the case for 4.0) there's no reason to keep it.

Reply:
Godot uses `-miphoneos-version-min=10.0` so it's not. I'll change the code then

Reply:
Yeah if you want to keep this PR a simple backport of `master` changes, and then make two new PRs for further deprecation fixes for `master` and `3.2`, that's fine with me. The comments made here can be used as actionable feedback for this follow-up PR.

Reply:
Yeah, I think that should work better